src/util-logopenfile.c

```diff
@@ -65,13 +65,17 @@ SCLogOpenUnixSocketFp(const char *path, int sock_type, int log_err)
     saun.sun_family = AF_UNIX;
     strlcpy(saun.sun_path, path, sizeof(saun.sun_path));
 
+    SCLogInfo("Log connecting to unix socket %s", path);
```

Comment:
these changes seem unrelated to pcap directory processing and should go into their own commit